prow/plank/controller.go

```diff
@@ -404,6 +406,9 @@ func (c *Controller) startPod(pj kube.ProwJob) (string, string, error) {
 }
 
 func (c *Controller) getBuildID(server, name string) (string, error) {
+	if server == "" {
+		return uuid.NewV1().String(), nil
```

Comment:
Can we keep buildIDs numeric and increasing using [snowflakes](https://github.com/bwmarrin/snowflake) instead of UUIDs? Example IDs would be `880199696671313921` and `880200147634491393`.